packages/cli/src/commands/bundle/buildBundle.ts

```diff
@@ -72,13 +94,14 @@ async function buildBundle(
     await output.save(bundle, args, logger.info);
 
     // Save the assets of the bundle
-    const outputAssets = await server.getAssets({
+    const outputAssets: AssetData[] = await server.getAssets({
       ...Server.DEFAULT_BUNDLE_OPTIONS,
       ...requestOpts,
       bundleType: 'todo',
     });
 
     // When we're done saving bundle output and the assets, we're done.
+    // @ts-ignore
```

Comment:
why ts ignore? can you leave a comment explaining?

Reply:
the value `args.platform` is actually seen as `string | undefined` however in the function `saveAssets()` it is required which causes type error... looking at the code it looks like the `args.platform` would be provided at all times which makes no sense to actually make it optional from the code as well which is why I suppress the warning but what do you think of it

Reply:
Definitely shouldn't be suppressed. This is possible failure point and we need to make sure args.platform is defined before passing it further

Reply:
I just checked in now the `args.platform` has a default value of `ios` which means making it options is actually not necessary